doc/nrf/app_dev/device_guides/thingy53/building_thingy53.rst

```diff
@@ -84,7 +84,7 @@ In such cases, you can program the Thingy:53 the same way as the nRF5340 DK.
 The external debug probe must support Arm Cortex-M33 (such as the nRF5340 DK).
 You need a 10-pin 2x5 socket-to-socket 1.27 mm IDC (:term:`Serial Wire Debug (SWD)`) JTAG cable to connect to the external debug probe.
 
-This method is supported when programming with :ref:`Visual Studio Code <thingy53_build_pgm_vscode>`, :ref:`command line <thingy53_build_pgm_command_line>`, or the `Programmer app <Programming Nordic Thingy:53_>`_ from nRF Connect for Desktop.
+This method is supported when programming with :ref:`Visual Studio Code <thingy53_build_pgm_vscode>`, :ref:`command line <thingy53_build_pgm_command_line>`, or the `Programmer app <prog_thingy_53>`_ from nRF Connect for Desktop.
```

Comment:
Same here as the previous comment. Line  104 and 203.

Reply:
fixed